apps/settings/locales/settings.en-US.properties

```diff
@@ -303,7 +303,7 @@ message-read-reports-details = Request read report for each message sent
 # force the proper directionality of parentheses at the end of the string
 # regardless of the direction of the surrounding layout. Remove them only
 # if the translation puts them between other text.
-mmsTitle.innerHTML= &#x202a;Multimedia Messaging (MMS)&#x202c;
+mmsTitle= &#x202a;Multimedia Messaging (MMS)&#x202c;
```

Comment:
so, I tried setting `unicode-bidi: isolate` on `<h2>` it and it didn't work for some reason. I decided to instead of that use `<bdi>` tag inside `<h2>` and that works well.